src/vodja/ManagedGame.java

```diff
@@ -308,4 +327,10 @@ else if (gameType == GameType.HUMHUM) {
 	 * @return An array of indices that the component occupies.
 	 */
 	public Index[] losingComponent() { return game.losingComponent(); }
+	public boolean isValid(Index idx) { return validity[idx.i()][idx.j()]; }
```

Comment:
zakaj nebi tu samo klical `game.isValid`?

Reply:
To, da se ti pokaže senca kamna, ko gleš z miško čez polje, kliče `isValid`. Za Go to preveri, da poteza ni samomorilska, kar je časovno nezanemarljivo. V tem primeru je zato senca malo neodzivna. Če bo povzročalo težave v algoritmu, lahko dodava, da se cela tabela `validity` izračuna samo, ko se izvede končna poteza.

Reply:
Ni mi všeč, da je taka stvar v vodji; če je pomembna samo za GUI, naj bo v GUI.
Zaenkrat naj bo.